#! /bin/sh /usr/share/dpatch/dpatch-run
## 05-leave-existant-php-ini.dpatch by  <mhershberger@intrahealth.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: don't overwrite php.ini

@DPATCH@

diff -ub a/lib/Apache/TestConfigPHP.pm~ a/lib/Apache/TestConfigPHP.pm
--- a/lib/Apache/TestConfigPHP.pm~
+++ b/lib/Apache/TestConfigPHP.pm
@@ -171,6 +171,7 @@
     my $file = catfile $dir, 'php.ini';
     my $log  = catfile $self->{vars}->{t_logs}, 'error_log';
 
+    if(! -f $file) {
     $self->gendir($dir);
     my $fh = $self->genfile($file);
 
@@ -180,6 +181,7 @@
     close $fh or die "close $file: $!";
 
     $self->clean_add_file($file);
+    }
 }
 
 sub configure_php_tests {

Diff finished.  Wed Feb  6 23:53:13 2008
